zproject/local_settings_template.py

```diff
@@ -45,27 +45,30 @@
 # SSO_APPEND_DOMAIN = "example.com")
 SSO_APPEND_DOMAIN = None
 
-# Configure the outgoing SMTP server below. The default configuration
-# is prepopulated for GMail servers.  Change EMAIL_HOST for other
-# hosts, or leave it unset or empty to skip sending email.  Note if
-# you are using a GMail account to send outgoing email, you will
-# likely need to configure that account as "less secure" here:
-# https://support.google.com/accounts/answer/6010255.
+# Configure the outgoing SMTP server below. You will need to complete
+# the following steps:
 #
-# With the exception of reading EMAIL_HOST_PASSWORD from the Zulip
-# secrets file, Zulip uses Django's standard EmailBackend, so if
-# you're having issues, you may want to search for documentation on
-# using your email provider with Django.
+# (1) Fill out the outgoing email sending configuration below.  You
+# can skip sending emails entirely by commenting out EMAIL_HOST.
```

Comment:
Oh, this answers my question about whether sending emails is required. I'd personally bump that up outside a list item b/c the set of 3 steps is irrelevant if I don't want to send steps & comment out EMAIL_HOST instead.